privilege/privileges/privileges_test.go

```diff
@@ -773,31 +691,6 @@ func TestUseDB(t *testing.T) {
 	require.Error(t, err)
 }
 
-// func TestRevokePrivileges(t *testing.T) {
```

Comment:
We can just keep it and try to un-comment this case in the future.

Reply:
https://github.com/pingcap/tidb/blob/6db372a10b48b4640a4722c0c91e02477984dc68/tests/integrationtest/t/privilege/privileges.test#L231C1-L256

It's already moved and in the last PR I just comment it and forget to clean up... so this PR I clean it up here.